### PR TITLE
cytoviewer citation update

### PR DESCRIPTION
**Pull request by lassedochreden**
Hi @SchulzDan, 

I updated the _cytoviewer_ citation and adjusted the corresponding text a little. 

Thought this would also be a good test run for us for **workflow maintenance**. 

I included @nilseling (one last time, hope this is okay) just to be sure that this is all correct.

### CODE REVIEW

File: 06-quality_control.Rmd [hunk 1]
```diff
@@ -395,7 +396,7 @@ samples or batches of samples. Observing potential staining differences can be
 crucial to assess data quality. We will use ridgeline visualizations to check
 differences in staining patterns:
 
-```{r ridges, message=FALSE, fig.width=7, fig.height=25}
+```{r ridges, message=FALSE, warning = FALSE, fig.width=7, fig.height=25}
```
Comment (nilseling) [inline 06-quality_control.Rmd:399]: Why do you suppress the warnings here? Is this anything we need to address?

**Review by nilseling — APPROVED** [review]
Looks good. I assume you didn't change much in the text other than formatting? For some reason large parts of the text are highlighted as changed.

File: 06-quality_control.Rmd (hunk 1 above)
Comment (lassedochreden) [reply 06-quality_control.Rmd:399]: Yes not sure why most of the text is highlighted. I only changed the first paragraph and the "interactive visualization" paragraph. 

RE warnings: It looked like this to me. 

Screen Shot 2024-01-05 at 11 11 00

Comment (nilseling) [reply 06-quality_control.Rmd:399]: Ah ok, I guess that comes from the switch from `size` to `linewidth`. Then it's fine to suppress the warnings.

Comment (lassedochreden) [reply 06-quality_control.Rmd:399]: Once merged, do I need to create a new `release`?

Comment (nilseling) [reply 06-quality_control.Rmd:399]: You could. It's only a minor change. So only bump the z version

**Comment by nilseling** [comment]
I saw now that the Hyperlink to interactive image visualization points to the interactive single-cell visualization. I guess you used the same # tag?

**Comment by lassedochreden** [comment]
Okay, will change the # tag (did not double-check whether this was used before; my fault) directly on main.

**Pull request merged**